src/components/WorkStyle/index.jsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import WorkStyleCard from '../WorkStyleCard/index';
+import Heading from '../Heading/index';
+import { charityAPI } from '../../clients';
+import './styles.css';
+
+export default class WorkStyle extends React.Component {
+  state = {
+    data: {},
+    loading: true,
+    error: false,
+    errorMessage: ''
+  };
+
+  componentDidMount() {
+    this.setState({ loading: true });
```

Comment:
you don't need this as the initial `state.loading` is also true. 

You may set the loading to true if you are calling the API multiple times, but then the code won't be in `componentDidMount`, it will be somewhere else.